reposilite-plugins/webhook-plugin/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 dzikoysk
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.reposilite.plugins"
+
+plugins {
+    id("com.github.johnrengelman.shadow") version "7.1.1"
+    kotlin("jvm")
+}
+
+application {
+    mainClass.set("com.reposilite.plugin.webhook.WebhookPluginKt")
+}
+
+dependencies {
+    compileOnly("net.dzikoysk:exposed-upsert:1.0.3")
+    compileOnly(project(":reposilite-backend"))
+    implementation("club.minnced:discord-webhooks:0.8.0")
```

Comment:
It’s easier to use a builder class rather than constructing a JSON request. I could use a json request if you’d like although the library is pretty small in size.

Reply:
Just wondering, in the end Discord impl should be moved to standalone plugin, so it's not that important.